packages/perseus/src/perseus-types.ts

```diff
@@ -1131,6 +1131,7 @@ export type PerseusMeasurerWidgetOptions = {
     rulerLength: number;
     // Containing area [width, height]
     box: [number, number];
+    // TODO(benchristel): static is not used. Remove it?
```

Comment:
I suspect we can. These types were originally generated by iterating over our `en` content and deriving TypeScript types from them. I suspect `static` shows up in these widget options due to bugs that existed for some amount of time in our editors but have since been fixed (or not).